servershr/Client.h

```diff
@@ -86,8 +86,8 @@ static inline int Client_free(Client *client, fd_set *fdactive)
     MDSDBG(CLIENT_PRI, CLIENT_VAR(client));
     if (client->reply_sock != INVALID_SOCKET)
     {
-      shutdown(client->reply_sock, 2);
-      close(client->reply_sock);
+      shutdown(client->reply_sock, SHUT_RDWR);
```

Comment:
Global question regarding `mdstcl` client and `mdsip` (and "action servers") . . .
- Are sockets always created with `SO_REUSEADDR` so they can be used again during `TIME_WAIT` status?

I'm not as familiar with the mdsip code as you.   My quick search of the source leads me to believe this is true, but I'm not 100% positive.

Reply:
The `SO_REUSEADDR` is a generic question, thus will be moved to a different forum.   Testing shows that this PR does not leak sockets during normal usage, and thus the `SO_REUSEADDR` question need not be answered at this time.